src/components/Patient/ShiftCreate.tsx

```diff
@@ -108,7 +110,7 @@ export const ShiftCreate = (props: patientShiftProps) => {
     errors: { ...initError },
   };
 
-  const { data: patientData } = useTanStackQueryInstead(routes.getPatient, {
+  const { data: _patientData } = useTanStackQueryInstead(routes.getPatient, {
```

Comment:
Switch to using `useQuery` instead. useTanStackQueryInstead is for legacy fn support, but we are slowly switching to useQuery (which uses tanstack).

Same applies for any post requests (switch to using `useMutate`), like in the POST req in this file.